apisix/plugins/log-rotate.lua

```diff
@@ -299,9 +303,9 @@ local function rotate()
     elseif max_size > 0 then
         local access_log_file_size = file_size(default_logs[DEFAULT_ACCESS_LOG_FILENAME].file)
         local error_log_file_size = file_size(default_logs[DEFAULT_ERROR_LOG_FILENAME].file)
-        local files = core.table.new(2, 0)
+        local files = {}
```

Comment:
why make this change? It seems unrelated to me. If it really is, you should remove it.

Reply:
Same that no need to pre-allocate. It only needs to store one filename when `enable_access_log` is false.